tests/migration_test.go

```diff
@@ -1252,6 +1253,197 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
+				cpuLines := vcpuPinOutputLines[2 : len(vcpuPinOutputLines)-2]
+
+				for _, line := range cpuLines {
+					lineSplits := strings.Split(line, " ")
+					var lineNoSpaces []string
+					for _, split := range lineSplits {
+						if split != "" {
+							lineNoSpaces = append(lineNoSpaces, split)
+						}
+					}
+
+					cpu, err := strconv.Atoi(lineNoSpaces[1])
+					Expect(err).To(BeNil(), "cpu id is non string in vcpupin output")
+
+					cpuSet = append(cpuSet, cpu)
+				}
+
+				return cpuSet
+			}
+
+			getLibvirtDomainCPUSet := func(vmi *v1.VirtualMachineInstance) []int {
+				vmiPod := tests.GetPodByVirtualMachineInstance(vmi)
+
+				stdout, stderr, err := tests.ExecuteCommandOnPodV2(virtClient,
+					vmiPod,
+					"compute",
+					[]string{"virsh", "vcpupin", fmt.Sprintf("%s_%s", vmi.GetNamespace(), vmi.GetName())})
+				Expect(stderr).To(BeEmpty())
+				Expect(err).To(BeNil())
+
+				return parseVCPUPinOutput(stdout)
+			}
+
+			parseSysCpuSet := func(cpuset string) []int {
+				strs := strings.Split(cpuset, ",")
+				ints := make([]int, len(strs))
+				for i, s := range strs {
+					ints[i], _ = strconv.Atoi(s)
+				}
+				return ints
+			}
+
+			getPodCPUSet := func(pod *k8sv1.Pod) []int {
+				stdout, stderr, err := tests.ExecuteCommandOnPodV2(virtClient,
+					pod,
+					"compute",
+					[]string{"cat", "/sys/fs/cgroup/cpuset/cpuset.cpus"})
+				Expect(stderr).To(BeEmpty())
+				Expect(err).To(BeNil())
+
+				return parseSysCpuSet(stdout)
+			}
+
+			getVirtLauncherCPUSet := func(vmi *v1.VirtualMachineInstance) []int {
+				vmiPod := tests.GetPodByVirtualMachineInstance(vmi)
+
+				return getPodCPUSet(vmiPod)
+			}
+
+			hasCommonCores := func(vmi *v1.VirtualMachineInstance, pod *k8sv1.Pod) bool {
+				set1 := getVirtLauncherCPUSet(vmi)
+				set2 := getPodCPUSet(pod)
+				for _, corei := range set1 {
+					for _, corej := range set2 {
+						if corei == corej {
+							return true
+						}
+					}
+				}
+				return false
+			}
+
+			BeforeEach(func() {
+				By("getting the list of worker nodes that have cpumanager enabled")
+				nodeList, err := virtClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{
+					LabelSelector: fmt.Sprintf("%s=,%s=%s", workerLabel, "cpumanager", "true"),
+				})
+				Expect(err).To(BeNil())
+				Expect(nodeList).ToNot(BeNil())
+				nodes = nodeList.Items
+				Expect(len(nodes)).To(BeNumerically(">=", 2), "at least two worker nodes with cpumanager are required for migration")
+
+				By("creating a migratable VMI with 2 dedicated CPU cores")
+				migratableVMI = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				migratableVMI.Spec.Domain.CPU = &v1.CPU{
+					Cores:                 uint32(2),
+					DedicatedCPUPlacement: true,
+				}
+				migratableVMI.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("512Mi")
+				// TODO? Other tests use only 128Mi + the following but my test cluster doesn't do hugepages apparently
```

Comment:
128Mi is a default in KubeVirtCI

Reply:
Right, but a VMI with dedicated cores, 128Mi of ram and no hugepages get OOM killed...
Since I can't do hugepages I bumped the RAM and it did the trick.